docs/cloud-native-security/kspm.asciidoc

```diff
@@ -1,2 +1,38 @@
 [[kspm]]
 = Kubernetes security posture management
+
+The Kubernetes Security Posture Management (KSPM) integration allows you to monitor how your Kubernetes clusters' configuration measures up to security benchmarks.
+
+To set up the integration, you'll need to first add it to an agent policy, then deploy the KSPM DaemonSet to the Kubernetes clusters you want to monitor.
+
+[discrete]
+== Set up a KSPM integration
+To install the integration:
+
+1. Go to *Dashboards -> Cloud Posture*
+2. Click *Add a CIS integration*.
+3. Click *Add Kubernetes Security Posture Management*.
+4. Name your integration.
+5. Select whether to use the CIS or EKS Benchmarks — use CIS unless you're deploying on EKS.
+6. Select the {agent} policy where you want to add the integration.
+7. Click *Save and continue*, then *Add agent to your hosts*.
+8. The *Add agent* wizard appears, and provides you a DaemonSet manifest `.yaml` with your `Fleet ID`, `Fleet URL`, and other configuration information pre-populated.
+
+The *Add agent* wizard will help you to deploy a DaemonSet on the Kubernetes clusters you wish to monitor. To do this, for each cluster:
+
+1. Download the manifest and make any necessary revisions to its configuration to suit the needs of your environment.
+2. Apply the manifest using the `kubectl apply -f ` command, for example: `kubectl apply -f elastic-agent-managed-kubernetes.yaml`
+
+After a minute, the *Add agent* wizard will show “Agent enrollment confirmed”, then “Incoming data confirmed”. You can then click *View assets* to see where the newly-collected configuration information appears throughout {kib}, including the <<findings-page,Findings page>> and the <<cloud-posture-dashboard, Cloud Posture dashboard>>.
+
+[[findings-page]]
+[discrete]
+== The Findings page
+
+image::images/findings-page.png[The Findings page]
```

Comment:
Add 1 px border to image. Also, is the search bar essential to include in the screenshot? We've cropped similar search bars out of other Security app screenshots, because it's typically a Kibana element that can change without notice.